public/views/editProfile.php

```diff
@@ -1,12 +1,66 @@
 <?php
-$SessionController = new SessionController;
+$SessionController = new SessionController();
+$userIsAuthenticated = $SessionController::isLogged();
+$userInfo = $SessionController->unserializeUser();
```

Comment:
Najpierw musimy sprawdzić czy użytkownik jest zalogowany, a dopiero później gdy to już wiemy to zaczytywać dane. Sprawdzenie znajduje się w liniach 6-8, a zaczytanie w 17, więc linijka po prostu do usunięcia.